### PR TITLE
MOO-394, MOO-395:Implement conditions widget to show diagnoses

### PR DESCRIPTION
**Pull request by Guimino**
- Remove DIAGNOSES widget in the Patient dashboard

- Added a changeset  to set the default locale to **'en'** (ISO 639-1),  this will resolve the problem in searching (concept, and other resources) using default locale.

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java [hunk 1]
```diff
@@ -67,7 +67,8 @@ private void installMSFMeta() {
         Context.getService(AppFrameworkService.class).disableApp(MSFCoreConfig.REGISTRATION_APP_EXTENSION_ID);
         Context.getService(AppFrameworkService.class).enableApp(MSFCoreConfig.MSF_REGISTRATION_APP_EXTENSION_ID);
 
-        // disable the default find patient app to provide one which allows searching for patients at the footer of the search for patients page
+        // disable the default find patient app to provide one which allows
```
Comment (kaweesi) [inline api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java]: any reason for splitting this comment

File: api/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -97,4 +97,10 @@
 			<column name="uuid" value="fc339d1c-b11b-11e8-96f8-529269fb1459" />
 		</insert>
 	</changeSet>
+	
+	<changeSet id="20180927-0910" author="guimino"> 
+		<comment>setting the default locale to en using the language code(ISO 639-1)</comment>
```
Comment (kaweesi) [inline api/src/main/resources/liquibase.xml]: use the api to set this, but u don't even need to do this at all, please revert this

File: api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java [hunk 3]
```diff
@@ -106,6 +107,9 @@ private void installMSFMeta() {
 
         log.info("Installing MSF Forms");
         installMsfForms();
+
+        log.info("Disabling diagnoses");
+        Context.getService(AppFrameworkService.class).disableApp(MSFCoreConfig.WGT_APP_DIAGNOSES_ID);
```
Comment (kaweesi) [inline api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java]: re-enable this app when tis module is stopped

File: api/src/main/resources/liquibase.xml (hunk 2 above)
Comment (Guimino) [reply api/src/main/resources/liquibase.xml]: to use the api you mean I a should set via the OpenMRS Adminstration menu?

File: api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java (hunk 3 above)
Comment (Guimino) [reply api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java]: re-enabled

File: api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java (hunk 1 above)
Comment (Guimino) [reply api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java]: reverted, was  problem caused by my eclipse save action configuration and/or eclipse formatting (ctr+shift+f)

File: api/src/main/resources/liquibase.xml (hunk 2 above)
Comment (kaweesi) [reply api/src/main/resources/liquibase.xml]: its definitely better to update it in the activator than via liquibase, what was the issues? i actually think that will be changed according to which country the installation is running and so we need to rather resolve the warning/error perhaps

Comment (kaweesi) [reply api/src/main/resources/liquibase.xml]: until this change does more than toggle that widget, am convinced handling this should not be part of this change

Comment (Guimino) [reply api/src/main/resources/liquibase.xml]: @kaweesi , I think now understood what you meant with using api to set this, will work in this changes

Comment (Guimino) [reply api/src/main/resources/liquibase.xml]: removed the changeset of update default locale and created a PR #26  to update it in the api

File: api/src/main/java/org/openmrs/module/msfcore/MSFCoreConfig.java [hunk 4]
```diff
@@ -351,4 +351,6 @@
     public final static String GP_SYNC_WITH_DHIS2 = "msfcore.syncWithDHISOnPatientRegistration";
 
     public final static String URL_POSTFIX_OPTIONSETS = "/api/optionSets.json?paging=false&fields=name,options[:code,name]";
+
+    public final static String WGT_APP_DIAGNOSES_ID = "coreapps.diagnoses";
```
Comment (kaweesi) [inline api/src/main/java/org/openmrs/module/msfcore/MSFCoreConfig.java]: have u considered using our current convention such as SEARCH_APP_EXTENSION_ID, MSF_SEARCH_APP_EXTENSION_ID, MSF_REGISTRATION_APP_EXTENSION_ID, REGISTRATION_APP_EXTENSION_ID?

Comment (Guimino) [reply api/src/main/java/org/openmrs/module/msfcore/MSFCoreConfig.java]: Done

**Pull request closed without merging**